### DISCUSSION TITLE
针对PDF是那种一页上面分成两竖列的PDF，识别时是按一行完成再识别第二行的这种情况有参数可以调整吗？

### DISCUSSION THREAD

**Discussion by tslxxcxy** [RapidAI/RapidOCRPDF#12] [Q&A] [answered]
![Snipaste_2024-03-30_21-18-13](https://github.com/RapidAI/RapidOCRPDF/assets/44352986/990ab732-843a-4615-8cce-e97f70f9e577)
如上图一样，它识别是每一行一行的识别出来的，但是我想能不能通过参数让它识别是按左右部分完成的

**Comment by SWHL** [comment]
这个暂时没有参数控制，需要自己后处理一下

**Comment by tslxxcxy** [comment]
好的，非常感谢，

**Comment by tslxxcxy** [comment]
你好，那这种问题，是不是也可以用你在issue，提过的就是处理页眉页脚的方法通过layout模块去识别一下pdf的版面行吗，或者说是在依托的RapidOCR上加参数，这种行不行

**Comment by SWHL** [comment]
可以的

**Comment by tslxxcxy** [comment]
试了。那个raplayout只能识别图片。如果我把PDF文件转为图片，可以识别，但是通过rapidocr识别这个转换为图片的pdf.错误率很大。可能要等你有时间把这个版面还原实现后就没问题了。不得不说大佬你这个识别扫描版的pdf是真的厉害。没有错误，但是转为图片后通过rapidocr识别就有错误。

**Reply by SWHL** [reply, marked as answer]
RapidOCRPDF这个库是两种思路实现的。有些PDF可以直接抽取其中文本，所以100%识别准确。如果不能直接抽取的，会有OCR，则会有错误。

**Reply by tslxxcxy** [reply]
这个问题我也有一个方法。就是针对左右两边的pdf，可以先切在识别。

**Reply by SWHL** [reply]
切分的话，针对简单版面PDF可以，如果针对复杂，像多栏的，且PDF背景不是纯色，效果就差了。
